odata-protocol/11 Data Service Requests.md

```diff
@@ -809,9 +812,14 @@ GET http://host/service/Products?$top=5
 The `$skip` system query option specifies a non-negative integer $n$ that
 excludes the first $n$ items of the queried collection from the result.
 
-Let $A$ be a copy of the input set with a total order that extends any existing order of the input set but is otherwise chosen by the service. The total order MUST be stable across requests.
+Let $A$ be a copy of the input set with a total order that extends any
+existing order of the input set but is otherwise chosen by the service. The
```

Comment:
Why no mentioning of `$orderby` here? I'd expect almost the same text here as in the section on `$top` even if repetition is a bit boring 😄